mobileapp/config/development.json

```diff
@@ -0,0 +1,21 @@
+{
+  "environment_name": "Development",
+  "backend": {
+    "ip": "localhost",
+    "port": 8080
```

Comment:
Since all of us will use slightly different config, try to make `development-example.json` (just like in backend), add this `development.json` to mobile app `.gitigonre` and add info about it to mobile app `README.MD`

Reply:
Done